Unable to run `./agent status`

### Overview

Running the Status command returns the following error:
```bash
# $ ./agent status
Getting the status from the agent.

panic: open /opt/datadog-agent/dist/templates/header.tmpl: no such file or directory

goroutine 1 [running]:
html/template.Must(...)
	/usr/local/go/src/html/template/template.go:372
github.com/DataDog/datadog-agent/pkg/status.renderHeader(0x9d4768, 0x12277a0, 0x12277c0)
	/go/src/github.com/DataDog/datadog-agent/pkg/status/render.go:112 +0x188
github.com/DataDog/datadog-agent/pkg/status.FormatStatus(0x12d4000, 0x1c40, 0x1e00, 0x1c40, 0x1e00, 0x0, 0x0)
	/go/src/github.com/DataDog/datadog-agent/pkg/status/render.go:52 +0x48c
github.com/DataDog/datadog-agent/cmd/agent/app.requestStatus(0x88bfeb, 0x4)
	/go/src/github.com/DataDog/datadog-agent/cmd/agent/app/status.go:108 +0x24c
github.com/DataDog/datadog-agent/cmd/agent/app.glob..func10(0xf6dbf8, 0xf83b2c, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/DataDog/datadog-agent/cmd/agent/app/status.go:60 +0x1bc
github.com/DataDog/datadog-agent/vendor/github.com/spf13/cobra.(*Command).execute(0xf6dbf8, 0xf83b2c, 0x0, 0x0, 0xf6dbf8, 0xf83b2c)
	/go/src/github.com/DataDog/datadog-agent/vendor/github.com/spf13/cobra/command.go:826 +0x34c
github.com/DataDog/datadog-agent/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xf6ca78, 0x42108, 0x7b13b8, 0x10000e0)
	/go/src/github.com/DataDog/datadog-agent/vendor/github.com/spf13/cobra/command.go:914 +0x230
github.com/DataDog/datadog-agent/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/go/src/github.com/DataDog/datadog-agent/vendor/github.com/spf13/cobra/command.go:864
main.main()
	/go/src/github.com/DataDog/datadog-agent/cmd/agent/main.go:18 +0x20
```

The `dist/` folder is missing from the main folder.

--- a/Dockerfile
+++ b/Dockerfile
@@ -26,4 +26,7 @@ RUN cd $AGENT_FOLDER && \
 CMD cd $AGENT_FOLDER && \
   GOOS=$GOOS GOARCH=$GOARCH GOARM=$GOARM invoke agent.build --puppy && \
   mv bin/agent/dist bin/agent/datadog-agent && \
+  mkdir bin/agent/dist -p && \
+  mv bin/agent/datadog-agent/templates bin/agent/dist && \
+  mv bin/agent/datadog-agent/views bin/agent/dist && \
   tar cvf ./target/agent-armv7-$VERSION.tar.gz -C bin/agent/ .

